sdk/storage/tests.yml

```diff
@@ -30,28 +31,7 @@ extends:
             Selection: sparse
             GenerateVMJobs: true
       EnvVars:
-        STORAGE_ACCOUNT_NAME: $(python-storage-storage-account-name)
-        STORAGE_ACCOUNT_KEY: $(python-storage-storage-account-key)
-        STORAGE_DATA_LAKE_ACCOUNT_NAME: $(python-storage-data-lake-account-name)
-        STORAGE_DATA_LAKE_ACCOUNT_KEY: $(python-storage-data-lake-account-key)
-        BLOB_STORAGE_ACCOUNT_NAME: $(python-storage-blob-storage-account-name)
-        BLOB_STORAGE_ACCOUNT_KEY: $(python-storage-blob-storage-account-key)
-        REMOTE_STORAGE_ACCOUNT_NAME: $(python-storage-remote-storage-account-name)
-        REMOTE_STORAGE_ACCOUNT_KEY: $(python-storage-remote-storage-account-key)
-        PREMIUM_STORAGE_ACCOUNT_NAME: $(python-storage-premium-storage-account-name)
-        PREMIUM_STORAGE_ACCOUNT_KEY: $(python-storage-premium-storage-account-key)
-        OAUTH_STORAGE_ACCOUNT_NAME: $(python-storage-oauth-storage-account-name)
-        OAUTH_STORAGE_ACCOUNT_KEY: $(python-storage-oauth-storage-account-key)
-        ACTIVE_DIRECTORY_APPLICATION_ID: $(aad-azure-sdk-test-client-id)
-        ACTIVE_DIRECTORY_APPLICATION_SECRET: $(aad-azure-sdk-test-client-secret)
-        ACTIVE_DIRECTORY_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
-        CONNECTION_STRING: $(python-storage-blob-connection-string)
-        BLOB_CONNECTION_STRING: $(python-storage-blob-connection-string)
-        PREMIUM_CONNECTION_STRING: $(python-storage-premium-connection-string)
         TEST_MODE: 'RunLiveNoRecord'
```

Comment:
You should be able to remove this, but no need to block on it (and it doesn't hurt to keep).